scripts/o3de/ExportScripts/export_source_ios_xcode.py

```diff
@@ -44,51 +64,81 @@ def export_ios_xcode_project(ctx: exp.O3DEScriptExportContext,
+                                  [ASSET_PLATFORM], validated_seedslist_paths, preprocessed_seedfile_paths, 
+                                  ctx.project_path/'AssetBundling', max_bundle_size,
+                                  logger)
+
+    if should_build_all_assets:
+        if (ctx.project_path / 'AssetBundling/Bundles').is_dir():
+            (ctx.project_path / 'Pak' / f'{ctx.project_name}_ios_paks').mkdir(parents=True)
+            exp.process_command(['cp', '-r', str(ctx.project_path / 'AssetBundling/Bundles/engine_ios.pak'), 
+                            str( ctx.project_path / 'Pak' / f'{ctx.project_name}_ios_paks')], cwd=ctx.project_path)
+            exp.process_command(['cp', '-r', str(ctx.project_path / 'AssetBundling/Bundles/game_ios.pak'), 
+                            str( ctx.project_path / 'Pak' / f'{ctx.project_name}_ios_paks')], cwd=ctx.project_path)
 
     # Generate the Xcode project file for the O3DE project
     cmake_toolchain_path = ctx.engine_path / 'cmake/Platform/iOS/Toolchain_ios.cmake'
 
     exp.process_command(['cmake', '-B', ios_build_folder_str, '-G', "Xcode", f'-DCMAKE_TOOLCHAIN_FILE={str(cmake_toolchain_path)}', '-DLY_MONOLITHIC_GAME=1'],
                     cwd= ctx.project_path)
 
-
     logger.info(f"Xcode project file should be generated now. Please check {ios_build_folder_str}")
 
+    xcode_build_common_args = ['xcodebuild', '-project', str(target_ios_project_path / f'{ctx.project_name}.xcodeproj'),
+                        '-scheme', f'{ctx.project_name}.GameLauncher', '-configuration', build_config, '-sdk',
+                        'iphoneos', '-allowProvisioningUpdates']
 
+    exp.process_command(xcode_build_common_args + ['clean'], cwd=ctx.project_path)
+    
+    exp.process_command(xcode_build_common_args + ['build'], cwd=ctx.project_path)
+    
+    # Note: This workaround enables the creation of an IPA file, even if xcode archive doesn't work, but will require manual upload
+    logger.info(f"Creating IPA file for {ctx.project_name}")
+    payload_path = target_ios_project_path / 'bin'/ build_config /'Payload'
+    payload_path_str = str(payload_path)
+    exp.process_command(['mkdir', payload_path_str], cwd=ctx.project_path)
+    exp.process_command(['cp', '-r', str(target_ios_project_path / 'bin'/ build_config / f'{ctx.project_name}.GameLauncher.app'), 
+                        str( payload_path / f'{ctx.project_name}.GameLauncher.app')], cwd=ctx.project_path)
+    
+    shutil.make_archive(payload_path, 'zip', root_dir=(payload_path.parent.resolve()))
+    ipa_path_str = str(target_ios_project_path / 'bin'/ build_config /f'{ctx.project_name}.GameLauncher.ipa')
+    exp.process_command(['mv', str(payload_path)+".zip", ipa_path_str])
 
-# This code is only run by the 'export-project' O3DE CLI command
-if "o3de_context" in globals():
-    global o3de_context
-    global o3de_logger
+    logger.info(f"iOS IPA file should be generated now. Please check {ipa_path_str}")
```

Comment:
I think you can check for them by using Path's .is_file() and returning an error if its not there (or even other validations if you could)